integration-templates/cal-com-v2/nango.yaml

```diff
@@ -0,0 +1,163 @@
+integrations:
+    cal-com-v2:
+        syncs:
+            events:
+                sync_type: full
+                runs: every hour
+                endpoint: GET /events
+                output: Event
+                description: Retrieve all upcoming events per a user
+            event-types:
+                sync_type: full
+                runs: every hour
+                endpoint: GET /event/types
```

Comment:
```suggestion
                endpoint: GET /events/types
```

Reply:
Will keep as is as it is matching the calendly endpoint.